tools/deploy_to_npm.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Warning: this file needs to work in Python 3.7 since it is used in the deploy
+# stage and the docker image cibuilds/github:0.13 is Alpine 3.10 which comes
+# with Python 3.7
+
+import subprocess
+import sys
+from pathlib import Path
+
+from bump_version import python_version_to_js_version
+
+VERSION = python_version_to_js_version(sys.argv[-1])
+IS_PRERELEASE = "a" in VERSION
+PKG = "pyodide"
+PKG = "@hoodmane/hoodmane-test-pyodide"
+
+cwd = Path("./dist")
+
+cmd = ["npm", "publish"]
+if IS_PRERELEASE:
+    cmd.extend(["--tag", "next"])
+result = subprocess.run(cmd, cwd=cwd)
+if result.returncode:
+    sys.exit(result.returncode)
+
+if not IS_PRERELEASE:
+    sys.exit(
+        subprocess.run(["npm", "dist-tag", "add", f"{PKG}@{VERSION}", "next"], cwd=cwd)
```

Comment:
I guess we don't want to add `next` tag to a stable release?

I think this was for:

```sh
# https://pyodide.org/en/0.19.1/development/maintainers.html#making-an-alpha-release
If you accidentally publish the alpha release over the stable latest tag, you can fix it with:
npm dist-tag add pyodide@a.b.c latest where a.b.c should be the latest stable version.
Then use npm dist-tag add pyodide@a.b.c-alpha.d next to set the next tag to point to
the just-published alpha release.
```

Reply:
There are two tags `latest` which is the latest stable release and `next` which is the latest release. So when we publish an alpha, we only update next and when we publish a stable release we update both.

Reply:
Okay, I was misunderstanding the docs. Thanks!